src/js/scaleworkshop.js

```diff
@@ -18,7 +18,6 @@ import {
   line_to_decimal,
   sanitize_filename,
 } from './helpers/converters.js'
-import { synth } from './synth.js'
```

Comment:
These were already removed in the latest `import-export` commit, please check, if you have the latest version merged on your branch.

Reply:
I believe this was caused by a bad merge on my part, sorry about that!